recipes/champ/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "champ" %}
+{% set version = "2.0.2" %}
+{% set sha256 = "2e24d8fea58bf65bb13ec646cbb7404c4fc780ca4c07d5eb03c9aca9cb1609bb" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/ce/25/b7cba31f1781a7ddbcafb8ff8e96357ffa98d41704ccf7e19bcfe928649a/champ-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 0
+  skip: True  # [win]
```

It's fine if this makes the Windows build fail:
```suggestion
  noarch: python
```